source/distributed/fully_distributed_tria.cc

```diff
@@ -60,7 +60,8 @@ namespace parallel
       settings = construction_data.settings;
 
       // set the smoothing properties
-      if (settings & construct_multigrid_hierarchy)
+      if (settings &
+          parallel::fullydistributed::Settings::construct_multigrid_hierarchy)
```

Comment:
Make this`construction::Settings` (same thing everywhere else). Don't use deprecated things in the library.